static/gen/ast_defs.pony

```diff
@@ -157,20 +157,19 @@ primitive ASTDefs
       .> has("block", "Expr")
     
     for name in [
-      "Not", "Neg", "NegUnsafe", "AddressOf", "DigestOf"
+      "Not"; "Neg"; "NegUnsafe"; "AddressOf"; "DigestOf"
     ].values() do
       g.def(name)
         .> in_union("UnaryOp", "Expr")
         .> with_type()
         .> has("expr", "Expr")
     end
     
-    for name in [
-      "Add", "AddUnsafe", "Sub", "SubUnsafe",
-      "Mul", "MulUnsafe", "Div", "DivUnsafe", "Mod", "ModUnsafe",
-      "LShift", "LShiftUnsafe", "RShift", "RShiftUnsafe",
-      "Is", "Isnt", "Eq", "NE", "LT", "LE", "GE", "GT", "And", "Or", "XOr"
-    ].values() do
+    for name in ["Add"; "AddUnsafe"; "Sub"; "SubUnsafe"; "Mul"
+                 "MulUnsafe"; "Div"; "DivUnsafe"; "Mod"
+                 "ModUnsafe"; "LShift"; "LShiftUnsafe"; "RShift"
+                 "RShiftUnsafe";"Is"; "Isnt"; "Eq"; "NE"; "LT"
+                 "LE"; "GE"; "GT"; "And"; "Or"; "XOr"].values() do
```

Comment:
FYI, you don't have to close a PR to make the change - if you push a new commit to the same branch, it will show up in the original PR.